assets/common/components/batch_deploy_model/spec.yaml

```diff
@@ -0,0 +1,191 @@
+    default: cpu-cluster
+    description: Name of the compute target to execute the batch scoring jobs on
+  
+  size:
+    type: string
+    optional: true
+    enum:
+      - Standard_DS1_v2
+      - Standard_DS2_v2
+      - Standard_DS3_v2
+      - Standard_DS4_v2
+      - Standard_DS5_v2
+      - Standard_F2s_v2
+      - Standard_F4s_v2
+      - Standard_F8s_v2
+      - Standard_F16s_v2
+      - Standard_F32s_v2
+      - Standard_F48s_v2
+      - Standard_F64s_v2
+      - Standard_F72s_v2
+      - Standard_FX24mds
+      - Standard_FX36mds
+      - Standard_FX48mds
+      - Standard_E2s_v3
+      - Standard_E4s_v3
+      - Standard_E8s_v3
+      - Standard_E16s_v3
+      - Standard_E32s_v3
+      - Standard_E48s_v3
+      - Standard_E64s_v3
+      - Standard_NC4as_T4_v3
+      - Standard_NC6s_v2
+      - Standard_NC6s_v3
+      - Standard_NC8as_T4_v3
+      - Standard_NC12s_v2
+      - Standard_NC12s_v3
+      - Standard_NC16as_T4_v3
+      - Standard_NC24s_v2
+      - Standard_NC24s_v3
+      - Standard_NC24rs_v3
+      - Standard_NC64as_T4_v3
+      - Standard_ND40rs_v2
+      - Standard_ND96asr_v4
+      - Standard_ND96amsr_A100_v4
+    default: Standard_NC24s_v3
+    description: Compute instance size to deploy model. Make sure that instance type is available and have enough quota available.
+
+  min_instances:
+    type: integer
+    optional: true
+    default: 0
+    description: Minimum number of instances of the compute cluster
+
+  max_instances:
+    type: integer
+    optional: true
+    default: 1
+    description: Maximum number of instances of the compute cluster
+
+  idle_time_before_scale_down:
+    type: integer
+    optional: true
+    default: 120
+    description: Node Idle Time before scaling down amlCompute
+
+  output_file_name:
+    type: string
+    optional: true
+    default: predictions.csv
+    description: Name of the batch scoring output file.
+
+  max_concurrency_per_instance:
+    type: integer
+    optional: true
+    default: 1
+    description: The maximum number of parallel scoring_script runs per instance.
+
+  error_threshold:
+    type: integer
+    optional: true
+    default: -1
+    description: The number of file failures that should be ignored.
+
+  max_retries:
+    type: integer
+    optional: true
+    default: 3
+    description: The maximum number of retries for a failed or timed-out mini batch.
+
+  timeout:
+    type: integer
+    optional: true
+    default: 500
+    description: The timeout in seconds for scoring a single mini batch.
+
+  logging_level:
+    type: string
+    optional: true
+    default: info
+    description: The log verbosity level.
+
+  mini_batch_size:
+    type: integer
+    optional: true
+    default: 10
+    description: The number of files the code_configuration.scoring_script can process in one run() call.
+
+  instance_count:
+    type: integer
+    optional: true
+    default: 1
+    description: The number of nodes to use for each batch scoring job.
```

Comment:
Please check this as well. This is used in batch scoring, and we can mention that explicitly. Also what if I provide 4 and compute has 2 nodes? Will it fail or will still progress with 2 nodes?

Reply:
It will fail